pkgs/applications/misc/hollywood/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, makeWrapper, lib, coreutils, apg, atop, bmon, cmatrix
+, pygments, moreutils, util-linux, jp2a, man, mplayer, openssh, tree, mlocate
+, findutils, ccze, ncurses, python3, wget, libcaca, newsboat, rsstail, w3m
+, ticker, tmux, byobu }:
+let
```

Comment:
You should put this `let ... in` immediately before `installPhase` instead of lost here above

Reply:
Did that